package.json

```diff
@@ -13,18 +12,19 @@
     ".node-version"
   ],
   "scripts": {
-    "start": "npm run build && open-rpc-generator generate -c open-rpc-generator-config.json",
-    "test": "npm run test:lint && npm run test:unit",
-    "test:unit": "jest --coverage --passWithNoTests",
-    "test:integration": "npm run build && open-rpc-generator generate -c test-generator-config.json",
-    "test:lint": "tslint -c tslint.json 'src/**/*.ts' --fix",
-    "build": "npm run build:clean && tsc && cp -r templates build/",
+    "build": "yarn build:clean && tsc && cp -r templates build/",
     "build:clean": "rm -rf build",
+    "lint": "yarn lint:eslint && yarn lint:misc --check",
+    "lint:eslint": "eslint . --cache --ext js,ts",
+    "lint:fix": "yarn lint:eslint --fix && yarn lint:misc --write",
+    "lint:misc": "prettier '**/*.json' '**/*.md' '!CHANGELOG.md' '**/*.yml' '!.yarnrc.yml' --ignore-path .gitignore --no-error-on-unmatched-pattern",
+    "start": "yarn build && open-rpc-generator generate -c open-rpc-generator-config.json",
+    "test": "yarn test:lint && npm run test:unit",
+    "test:integration": "yarn build && open-rpc-generator generate -c test-generator-config.json",
```

Comment:
Was this meant to run on CI? It's not part of `test` so it doesn't currently get run.